static/malawi/config.py

```diff
@@ -259,11 +259,44 @@ class Messages(object):
                 "'help codes' for a list of commodity codes; " + \
                 "'help start' or 'help stop' to start and stop reminders."
 
+    # "rm" keyword
+    # Responses to Facility user reporting the refrigerator malfunction
+    FRIDGE_FACILITY_NO_GAS = "Thank you. The district will be informed that your refrigerator is out of gas."
+    FRIDGE_FACILITY_POWER_FAILURE = "Thank you. The district will be informed that your refrigerator has a power failure."
+    FRIDGE_FACILITY_BREAKDOWN = "Thank you. The district will be informed that your refrigerator is not working."
+
+    # Message to the District user notifying of the refrigerator malfunction
+    FRIDGE_DISTRICT_NO_GAS = "%(facility)s has refrigerator malfunction due to no gas."
```

Comment:
like I mentioned in the gdoc, we should probably include the facility code here. We might want to say something like like "respond with transfer %(code)s [to facility]" at the end of the message.